service/controller/key/common.go

```diff
@@ -57,3 +58,7 @@ func ReleaseName(releaseVersion string) string {
 func ReleaseVersion(getter LabelsGetter) string {
 	return getter.GetLabels()[label.ReleaseVersion]
 }
+
+func IsBundle(appName string) bool {
+	return strings.HasSuffix(appName, "-bundle")
```

Comment:
Will this support the ` security-pack` bundle? It does not have a matching suffix but it is still an app bundle.

Reply:
Same remarks as in cluster-apps-operator. No but the security pack should be named bundle as per the product decision.
Also, this is only for default apps and the security pack is not one of them

Reply:
I missed that one, thanks for clarifying.